plugins/outputs/orionres/processor/process.go

```diff
@@ -73,13 +55,50 @@ func (p *Processor) Process(ctx context.Context, metrics []telegraf.Metric) erro
 	return nil
 }
 
+func (p *Processor) addMetric(c *SessionClient, dbw *xv1.DatabaseWrite, updatedDefs map[*ResultDef]bool, metric telegraf.Metric) {
+	name := metric.Name()
+	//  t := metric.Type()
+	//  if t == telegraf.Counter {
+	//      counters and gauges are seperate metrics, use separate resultdef to avoid timestamp merges
+	//	name += counterSuffix
+	//  }
+	resultDef, ok := c.ResultDefs[name]
+	if !ok {
+		log.Printf("D! Creating result def %s", name)
+		resultDef = newResultDef(name, &p.MetricDefs)
+		c.ResultDefs[name] = resultDef
+		if !p.AddNewMetrics {
+			if len(resultDef.ResFacts) == 0 {
+				return
+			}
+		}
+		updatedDefs[resultDef] = true
+	}
+	if !resultDef.Enabled {
+		return
+	}
+	if p.AddNewMetrics {
+		if _, ok = updatedDefs[resultDef]; ok {
+			updateResultDef(resultDef, metric)
+		}
+	}
+	ds, rs := p.processMetric(c, resultDef, metric)
+	if rs != nil {
+		dbw.ResultSets = append(dbw.ResultSets, *rs)
+	}
+	for j := range ds {
+		dbw.DimensionSets = append(dbw.DimensionSets, *ds[j])
+	}
+}
+
 func (p *Processor) processMetric(c *SessionClient, r *ResultDef, metric telegraf.Metric) ([]*xv1.DimensionSetWrite, *xv1.ResultSetWrite) {
 	var ds []*xv1.DimensionSetWrite
+	maxCols := 2 + len(metric.Fields())
 	rs := &xv1.ResultSetWrite{
-		Name: r.Name,
+		Name:    r.Name,
+		Columns: make([]string, 0, maxCols),
 	}
-	var values []interface{}
-
+	values := make([]interface{}, 0, maxCols)
```

Comment:
Can you check the code alignment within this function?  Looks like a possible mixture of tabs and spaces.

Reply:
I think tabs and spaces are used but gofmt/goimports is the one adding them.